packages/wagmi/src/hooks/useDidMountEffect.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useRef } from 'react'
+
+const useDidMountEffect = (func: () => void, checkObject: any) => {
```

Comment:
how about we just copy the hooks from react use, and move to our hooks pkg?

https://github.com/streamich/react-use/blob/master/src/useUpdateEffect.ts

Reply:
yup I think the link I posted is what we want. I skips the first mount and only get called on update

Reply:
hmm ok i see your point